### PR TITLE
Added check for isModuleOutputEnabled('Mage_Review') in customer's backend page

### PR DESCRIPTION
**Pull request by eneiasramos**
Added validation for module Mage_Review in customer management ...

### CODE REVIEW

**Comment by ADDISON74** [comment]
Any steps to reproduce the issue?

**Comment by ADDISON74** [comment]
Any steps or screenshots to understand the issue?

**Comment by eneiasramos** [comment]
@ADDISON74 

The module Mage_Review is disabled. However, the link to it still appears. And when I click on it it generates an error.

image

**Comment by ADDISON74** [comment]
**VARIANT 1**
In order to disable the Review module I edited the file **/app/etc/modules/Mage_Review.xml**, but the Rating module depends on it, so I disable it too.

If I am accessing a customer account in the Backend, the [Product Reviews] tab is there. If I click on it I am seeing the fatal error bellow

fatal_error

**VARIANT 2**
Disabling the Review module output in the **Backend > System > Configuration > Advanced**, in the customer account page the **[Product Reviews]** tab is there and loads all reviews. 

Both variants are producing wrong behaviors related to the Review module. Now let's test this PR with both variants:

**Variant 1**: Disabling the module by editing the xml file the **[Product Reviews]** tab doesn't appear anymore. This PR fixes the issue.

**Variant 2**: Disabling the output of the module, the [Product Reviews] tab should not be visible in the customer account, but it is still there. **If I am not wrong this needs a fix too**.

File: app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tabs.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ protected function _beforeToHtml()
                 ]);
             }
 
-            if (Mage::getSingleton('admin/session')->isAllowed('catalog/reviews_ratings')) {
+            if (Mage::helper('core')->isModuleEnabled('Mage_Review') && Mage::getSingleton('admin/session')->isAllowed('catalog/reviews_ratings')) {
```
Comment (fballiano) [inline app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tabs.php:85]:
```suggestion
            if (Mage::helper('core')->isModuleOutputEnabled('Mage_Review') && Mage::getSingleton('admin/session')->isAllowed('catalog/reviews_ratings')) {
```

as @kiatng said

**Review by fballiano — APPROVED** [review]
thanks to @kiatng suggestion it works for all cases now

**Comment by ADDISON74** [comment]
With the last change both scenarios mention in my post are solved. We can merge it.

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**